tests/multi_ptr/multi_ptr_local_accessor_constructor.h

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for multi_ptr local_accessor constructors
+//
+*******************************************************************************/
+#ifndef __SYCLCTS_TESTS_MULTI_PTR_LOCAL_ACCESSOR_CONSTRUCTORS_H
+#define __SYCLCTS_TESTS_MULTI_PTR_LOCAL_ACCESSOR_CONSTRUCTORS_H
+
+#include "../common/common.h"
+#include "../common/section_name_builder.h"
+#include "multi_ptr_common.h"
+
+namespace multi_ptr_local_accessor_constructors {
+
+constexpr int expected_val = 42;
+
+template <typename T, typename Address_SpaceT, typename DimensionT>
+class run_local_accessor_cnstr_tests {
+  static constexpr sycl::access::address_space Space = Address_SpaceT::value;
+  static constexpr int dims = DimensionT::value;
```

Comment:
`Space` starts from the capital letter, but `dims` starts from the lowercase letter.  should they follow the one variable naming style?

Reply:
Changed to lowercase.